Makefile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 The Prometheus Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include Makefile.common
+
+.PHONY: get_dep
+getdep:
```

Comment:
That's spelled different here than anywhere else (`get_dep`). In different news, could you turn this into a real (Make-style) dependency of the Make targets that depend on getting the dependencies? (Must go deeper...)

I'm not a Make guru myself, but I guess there is some way to override the dependency structure here in `Makefile` to add dependencies to `Makefile.common`. (In doubt, let's page @matthiasr .)